### ISSUE TITLE
Plugin doesn't work?

### ISSUE THREAD

**Issue by zlylupaf** [ludovicchabant/obsidian-remember-file-state#26]
Hi, the state doesn't seem saved, I've tried restarting obsidian, saving the file, etc
I'm on latest obsidian version, windows 11

**Comment by Aetherinox** [comment]
Having the same issue. Plugin hasn't been updated in six months, and Obsidian has released updates since then. Could be dead.

**Comment by huachuman** [comment]
Yeah it's not remembering anything... anyone know what changed in the new obsidian that stopped this from working?

I'm really tired of constantly having to fold headings again every time I load a note

**Comment by kassirh** [comment]
Not sure if this might help. https://github.com/ludovicchabant/obsidian-remember-file-state/issues/28#issuecomment-3694800923
